client/scss/utilities/_variables.scss

```diff
@@ -1,12 +1,156 @@
+    'font-family': $wellcome-bold,
+    'font-size': 60px,
+    'letter-spacing': -20,
+    'line-height': 78px
+  ),
+  'WB3': (
+    'font-family': $wellcome-bold,
+    'font-size': 44px,
+    'letter-spacing': 0,
+    'line-height': 60px
+  ),
+  'WB4': (
+    'font-family': $wellcome-bold,
+    'font-size': 33px,
+    'letter-spacing': 0,
+    'line-height': 48px
+  ),
+  'WB5': (
+    'font-family': $wellcome-bold,
+    'font-size': 22px,
+    'letter-spacing': 20,
+    'line-height': 32px
+  ),
+  'WB6': (
+    'font-family': $wellcome-bold,
+    'font-size': 19px,
+    'letter-spacing': 20,
+    'line-height': 28px
+  ),
+  'WB7': (
+    'font-family': $wellcome-bold,
+    'font-size': 15px,
+    'letter-spacing': 20,
+    'line-height': 24px
+  ),
+  'HNM1': (
+    'font-family': $helvetica-neue-medium,
+    'font-size': 36px,
+    'letter-spacing': 20,
+    'line-height': 48px
+  ),
+  'HNM2': (
+    'font-family': $helvetica-neue-medium,
+    'font-size': 26px,
+    'letter-spacing': 20,
+    'line-height': 38px
+  ),
+  'HNM3': (
+    'font-family': $helvetica-neue-medium,
+    'font-size': 20px,
+    'letter-spacing': 40,
+    'line-height': 32px
+  ),
+  'HNM4': (
+    'font-family': $helvetica-neue-medium,
+    'font-size': 16px,
+    'letter-spacing': 40,
+    'line-height': 24px
+  ),
+  'HNM5': (
+    'font-family': $helvetica-neue-medium,
+    'font-size': 14px,
+    'letter-spacing': 80,
+    'line-height': 24px
+  ),
+  'HNM6': (
+    'font-family': $helvetica-neue-medium,
+    'font-size': 12px,
+    'letter-spacing': 80,
+    'line-height': 18px
+  ),
+  'HNL1': (
+    'font-family': $helvetica-neue-light,
+    'font-size': 36px,
+    'letter-spacing': 40,
+    'line-height': 48px
+  ),
+  'HNL2': (
+    'font-family': $helvetica-neue-light,
+    'font-size': 26px,
+    'letter-spacing': 40,
+    'line-height': 38px
+  ),
+  'HNL3': (
+    'font-family': $helvetica-neue-light,
+    'font-size': 20px,
+    'letter-spacing': 40,
+    'line-height': 32px
+  ),
+  'HNL4': (
+    'font-family': $helvetica-neue-light,
+    'font-size': 16px,
+    'letter-spacing': 40,
+    'line-height': 24px
+  ),
+  'HNL5': (
+    'font-family': $helvetica-neue-light,
+    'font-size': 14px,
+    'letter-spacing': 80,
+    'line-height': 24px
+  ),
+  'HNL6': (
+    'font-family': $helvetica-neue-light,
+    'font-size': 14px,
+    'letter-spacing': 40,
+    'line-height': 24px
+  ),
+  'HNL7': (
+    'font-family': $helvetica-neue-light,
+    'font-size': 12px,
+    'letter-spacing': 80,
+    'line-height': 18px
+  ),
+  'LR1': (
+    'font-family': $lettera-regular,
+    'font-size': 18px,
+    'letter-spacing': 0,
+    'line-height': 24px
+  ),
+  'LR2': (
+    'font-family': $lettera-regular,
+    'font-size': 14px,
+    'letter-spacing': -40,
+    'line-height': 24px
+  ),
+  'LR3': (
+    'font-family': $lettera-regular,
+    'font-size': 12px,
+    'letter-spacing': -20,
+    'line-height': 18px
+  )
+);
+
+$small-column-number: 4;
```

Comment:
Don't think we're using these anymore (I replaced them with the `$columns` map below).